wcfsetup/install/files/acp/templates/lineBreakSeparatedTextOptionType.tpl

```diff
@@ -0,0 +1,26 @@
+<ul class="scrollableCheckboxList" {*
```

Comment:
I would prefer augmenting an existing textarea similar to how Redactor / CodeMirror works over building a large DOM server side and dynamically adding the textarea. It's more fragile, because you need to keep the TypeScript + HTML synced. If you build everything in JS it's automatically consistent.

Reply:
This behavior is consistent with `Ui/ItemList`.